### PR TITLE
add tck test cases for session

### PR DESCRIPTION
**Pull request by HarrisChu**
#### What type of PR is this?
- [ ] bug
- [ ] feature
- [x] enhancement

#### What does this PR do?

add tck test cases for session

### CODE REVIEW

File: tests/common/utils.py [hunk 1]
```diff
@@ -441,3 +442,11 @@ def get_conn_pool(host: str, port: int):
     if not pool.init([(host, port)], config):
         raise Exception("Fail to init connection pool.")
     return pool
+
+def parse_service_index(name: str):
+    name = name.lower()
+    pattern = r"(graphd|storaged|metad)\[(\d+)\]"
```
Comment (jievince) [inline tests/common/utils.py:448]: `\[` ?

Comment (HarrisChu) [reply tests/common/utils.py:448]: match graphd[0]

File: tests/tck/features/admin/Sessions.feature [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      | /\d+/     | "root"   | ""        | /.*/       | /.*/       | /.*/      | 0        | "::ffff:127.0.0.1" |
+    When executing query:
+      """
+      CREATE USER user1 WITH PASSWORD 'nebula1';
+      CREATE SPACE s1(vid_type=int);
+      USE s1;
+      """
+    Then the execution should be successful
+    And wait 1 seconds
+    When login graphd[1] with user1 and nebula1
```
Comment (Shylock-Hg) [inline tests/tck/features/admin/Sessions.feature:25]: login "graphd[1" with "user" and "nebula1"

Comment (HarrisChu) [reply tests/tck/features/admin/Sessions.feature:25]: meaningful

**Review by Aiee — APPROVED** [review]
Good job!

**Review by Aiee — DISMISSED** [review]

File: tests/tck/conftest.py [hunk 3]
```diff
@@ -562,6 +578,19 @@ def result_should_contain(request, result, graph_spaces):
     )
 
 
+@then(parse("the result should contain, replace the holders with cluster info:\n{result}"))
```
Comment (Shylock-Hg) [inline tests/tck/conftest.py:581]: I think you extend it to more general, not just cluster info but anything could eval by python runtime.

Comment (HarrisChu) [reply tests/tck/conftest.py:581]: agreed, but I'm not sure what else should be add except cluster info.
we could change it in future if we have such cases?

Comment (Shylock-Hg) [reply tests/tck/conftest.py:581]: ok

**Review by Shylock-Hg — APPROVED** [review]

**Review by Aiee — DISMISSED** [review]

**Review by Shylock-Hg — APPROVED** [review]

**Review by Aiee — APPROVED** [review]

**Review by jievince — APPROVED** [review]
LGTM

**Pull request merged**